### ISSUE TITLE
duplicate entries 

### ISSUE THREAD

**Issue by Anusharuby** [PhaedrusTheGreek/logstash-input-nagioscheck#2]
Hi there,

I managed configure logstash-input-nagioscheck and get data from multiple hosts.

my first document:

{
  "_index": "nagios-checks-2021.02.02",
  "_type": "_doc",
  "_id": "bAOrY3cBS77U80IDFlfW",
  "_version": 1,
  "_score": null,
  "_source": {
    "took_ms": 4074.715,
    "message": "PING OK - Packet loss = 0%, RTA = 1.31 ms",
    "@version": "1",
    "status": "OK",
    "host": "ceo-frontend003",
    "check_uuid": "89db24d8-c6f5-40f6-ab09-6ec4024b397f",
    "@timestamp": "2021-02-02T16:54:52.821Z",
    "name": "Ping_check on logstash_ceo-frontend003",
    "type": "nagioscheck",
    "status_code": 0,
    "command": "/usr/local/nagios/libexec/check_ping -H 'ceo-frontend003' -w 85,30% -c 100,40%"
  },
  "fields": {
    "@timestamp": [
      "2021-02-02T16:54:52.821Z"
    ]
  },
  "sort": [
    1612284892821
  ]
}

second document:

{
  "_index": "nagios-checks-2021.02.02",
  "_type": "_doc",
  "_id": "awOrY3cBS77U80IDFlfW",
  "_version": 1,
  "_score": null,
  "_source": {
    "warning": 30,
    "label": "pl",
    "tags": [
      "_grokparsefailure"
    ],
    "critical": 40,
    "max": 0,
    "@version": "1",
    "check_uuid": "89db24d8-c6f5-40f6-ab09-6ec4024b397f",
    "uom": "%",
    "value": 0,
    "@timestamp": "2021-02-02T16:54:52.821Z",
    "name": "Ping_check on logstash_ceo-frontend003",
    "type": "nagiosmetric",
    "min": 0
  },
  "fields": {
    "@timestamp": [
      "2021-02-02T16:54:52.821Z"
    ]
  },
  "sort": [
    1612284892821
  ]
}

any help ? I am only interested in document 1
